csharp/src/Ice/Internal/EndpointI.cs

```diff
@@ -13,11 +13,6 @@ public interface EndpointI_connectors
 public abstract class EndpointI : Ice.Endpoint, IComparable<EndpointI>
 {
     public override string ToString()
-    {
-        return ice_toString_();
-    }
-
-    public virtual string ice_toString_()
```

Comment:
Also removed all the `ice_toString_` methods. You should obviously use ToString()!